examples/saving_collection.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+
+sys.path.insert(0, os.getenv("BUILDTEST_ROOT"))
```

Comment:
We probably don't need this envar anymore, if it's installed on the Python path in site-packages.